etna/libs/ts2vec/ts2vec.py

```diff
@@ -95,16 +95,18 @@ def fit(self, train_data, n_epochs=None, n_iters=None, verbose=False):
 
         loss_log = []
 
+        cur_epoch = 0
```

Comment:
- I think we should add license to each file of the ts2vec we took.
- Let's write what we changed in the original code below the copy of the license.